### PR TITLE
adding error page

### LINKED ISSUE #15: 404 Page
### Description
Design a 404 Not Found Component
 - Be creative and make it look geeky and codey :p

### Screenshots/testimonials
<place your screenshots or testimonials here if applicable>

### PR DESCRIPTION
**Pull request by sandesh-08**
fixes #15 
<Do not delete this template>
Description:  I have deleted the "not found  component" and added an Error component .

#### Checklist
- [ ] I've followed the Contributing guidelines provided in the repository.
- [ ] I've made the changes which were demanded in the linked issue.
- [ ] I've tested my code on a Chromium based browser.
- [ ] I've tested my code on Mozilla Firefox.
- [ ] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
<provide a relevant screenshot or any testimonial for the solved issue if applicable>
![Screenshot (56)] (https://user-images.githubusercontent.com/67503294/92992321-b6e10d80-f507-11ea-99d6-5f6d5b8c5631.png)

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ankiiitraj) [inline package-lock.json:1]: this file is not required
delete this file 
and while pushing first pull the changes

File: client/src/components/Error/Error.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './Error.css'
+
+const Error=()=>{
+    return(
+        <div className="container pt-5 mt-5">
+            <hr />
+            <div className="text-center text-danger firstdiv">
```
Comment (ankiiitraj) [inline client/src/components/Error/Error.js:10]: @EmperorYP7  Have a look. Do classes have consistent naming conventions?
LGTM

Comment (EmperorYP7) [reply client/src/components/Error/Error.js:10]: The uppermost div should be given a `className = "error-component container"`. We wont be using `pt` and `mt` or any such classes provided by bootstrap. Margins and padding should be handled manually in the corresponding CSS.

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
The repository has been updated since you created this PR. Kindly pull all the changes from the upstream and then we're good to go!

File: client/package-lock.json [hunk 3]
```diff
@@ -1,15797 +0,0 @@
-{
```
Comment (EmperorYP7) [inline client/package-lock.json:1]: Are we deleting `package-lock.json` for real @ankiiitraj? We'll be using yarn completely?

File: client/src/App.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ function App() {
           <Route path="/event" exact component={Event} />
           <Route path="/community" exact component={Community} />
           <Route path="/" exact component={Home} />
-          <Route path="/" component={NotFound} />
+          <Route component={Error} />
```
Comment (EmperorYP7) [inline client/src/App.js:24]: You may change this line to `<Route path = "/" component = {Error} />`

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]
Just a very minor tweak, and we're good to go!

File: client/package-lock.json (hunk 3 above)
Comment (ankiiitraj) [reply client/package-lock.json:1]: I didn't asked to delete client/package-lock.json but the package-lock.json in the main folder only.
Nonetheless @sandesh-08 add this file back. 
And not will not be using yarn in this project.

Comment (EmperorYP7) [reply client/package-lock.json:1]: Great that we got everything cleared now. So @sandesh-08, you know what to do! 😉

**Review by ankiiitraj — APPROVED** [review]

**Comment by ankiiitraj** [comment]
LGTM

File: client/src/App.js (hunk 4 above)
Comment (EmperorYP7) [reply client/src/App.js:24]: I'll fix this in #34.

**Review by EmperorYP7 — APPROVED** [review]
LGTM! Well done @sandesh-08! 🥳

**Pull request merged**